pages/draft-06/json-hyper-schema-release-notes.md

```diff
@@ -1,107 +1,104 @@
----
-title: JSON Hyper-Schema Draft-06 Release Notes
-section: docs
----
+### JSON Hyper-Schema Draft-06 Release Notes
 
 Release notes for migrating from draft-luff-json-hyper-schema-00 (draft-04) to draft-wright-json-schema-hyperschema-01 (draft-06).
 
 <div className="text-red-500 text-2xl mb-1"><b>NOTE</b>: draft-07 has been released</div>
 
-The [migration notes for draft-07](../draft-07/json-hyper-schema-release-notes) give a much more straightforward overview of migrating from draft-04 to draft-07 by skipping the complicated intermediate states of draft-05 and draft-06.  This page has been retained for historical interest, but it is not recommened for those who just want to get going with the latest draft.
+The [migration notes for draft-07](../draft-07/json-hyper-schema-release-notes) give a much more straightforward overview of migrating from draft-04 to draft-07 by skipping the complicated intermediate states of draft-05 and draft-06. This page has been retained for historical interest, but it is not recommened for those who just want to get going with the latest draft.
 
 **For implementors:** We recommend just implementing draft-07, and not draft-06 or earlier.
```

Comment:
suggestion: remove "just" in this sentence

Reply:
Done.